packages/documentation/src/sidebar.js

```diff
@@ -74,6 +74,7 @@ module.exports = {
       items: [
         {
           name: 'Introduction',
+          href: 'forms/when-to-use-fs'
```

Comment:
Just going to put this here for now. we can expand this document or reorganize it during an edit phase.